src/index.ts

```diff
@@ -191,6 +187,43 @@ client.on('voiceStateUpdate', async (oldState, newState) => {
   }
 });
 
+// Event triggered when a user's presence (e.g. status, activity, etc.) is changed.
+client.on("presenceUpdate", async (_, newPresence) => {
+  // Determine if this presence update is because a user is now playing a game
+  if (newPresence.activities[0]?.type === ActivityType.Playing) {
+    // Make sure the game is League of Legends
+    if (newPresence.activities[0].name === "League of Legends") {
```

Comment:
I might store this check in a variable just to reduce complexity in the switch.

I would suggest using a typescript enum for the switch if possible, similar to:

```      
enum LoLClientEvent {
  GAME_START = 'GameStart',
  FIRST_BLOOD = 'FirstBlood',
  CHAMPION_KILL = 'ChampionKill',
  MULTI_KILL = 'Multikill',
  ACE = 'Ace',
  GAME_END = 'GameEnd'
}

let newEvent = currentEvents[currentEvents.length - 1];

      if (newEvent) {
        switch (newEvent?.EventName) {
          case LoLClientEvent.GAME_START: {
            console.log('game start!');
            await playClip('halo_slayer.mp3', channel, audioPlayer);
            break;
          }
          case LoLClientEvent.FIRST_BLOOD: {
            console.log('First blood!');
            await playClip('firstblood.mp3', channel, audioPlayer);
            break;
          }
          case LoLClientEvent.CHAMPION_KILL: {
            console.log('kill occured!');
            await playClip('PUNCH.mp3', channel, audioPlayer);
            break;
          }
          case LoLClientEvent.MULTI_KILL: {```

Reply:
Fixed.